league/oauth2-server-bundle/0.4/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "League\\Bundle\\OAuth2ServerBundle\\LeagueOAuth2ServerBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "OAUTH_PRIVATE_KEY": "%kernel.project_dir%/%CONFIG_DIR%/jwt/private.pem",
+        "OAUTH_PUBLIC_KEY": "%kernel.project_dir%/%CONFIG_DIR%/jwt/public.pem",
```

Comment:
These are paths that will sometimes be different from one machine to another (hence using env vars)?